src/appLoader.js

```diff
@@ -14,7 +14,6 @@ const rootElement = document.getElementById('root');
 
 RModal.defaultStyles = { overlay: {}, content: {} };
 RModal.setAppElement(rootElement);
-window.SATURN_VERSION = import.meta.env.VITE_APP_VERSION;
```

Comment:
This seemed unnecessary, especially since now there will be an easier way to retrieve the version with `curl https://app.terra.bio/build-info.json`.